### PR TITLE
adding package.json 

### PR DESCRIPTION
**Pull request by mihirm21**
## This PR

- adds package.json to the repo for running the workflow of assigning reviewers

### Related Issues

Fixes #66 

### How to test

This workflow can be tested by editing or opening an issue

### CODE REVIEW

**Comment by mihirm21** [comment]
@beeme1mr

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/open-feature/ofep.git"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (beeme1mr) [inline package.json:14]:
```suggestion
  "license": "Apache-2.0",
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "ofep",
+  "version": "1.0.0",
+  "description": "A focal point for OpenFeature research, proposals and requests for comments",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (beeme1mr) [inline package.json:7]: Please remove this script

File: package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "ofep",
+  "version": "1.0.0",
+  "description": "A focal point for OpenFeature research, proposals and requests for comments",
+  "main": "index.js",
```
Comment (beeme1mr) [inline package.json:5]: Remove this line

File: package.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (beeme1mr) [inline package.json:1]: Please add the `octokit` dependency used in the GitHub action as a package dependency.

Comment (mihirm21) [reply package.json:1]: added. don't know why it was working for my project without it.

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (beeme1mr) [inline .vscode/settings.json:1]: Why was this added?

File: package.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  "name": "ofep",
+  "version": "1.0.0",
+  "description": "A focal point for OpenFeature research, proposals and requests for comments",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/open-feature/ofep.git"
+  },
+  "dependencies": {
+    "@octokit/rest": "^19.0.13"
```
Comment (beeme1mr) [inline package.json:10]: You can remove the explicit `octokit` install from the GitHub action now.

File: .vscode/settings.json (hunk 5 above)
Comment (mihirm21) [reply .vscode/settings.json:1]: Without this, the package.json file gave an error in vscode. The error was:
"["Problems loading reference 'vscode://schemas/settings/configurationDefaults': Unable to load schema from 'vscode://schemas/settings/configurationDefaults': cannot open vscode://schemas/settings/configurationDefaults. Detail: Unable to resolve text model content for resource vscode://schemas/settings/configurationDefaults."]"

Comment (mihirm21) [reply .vscode/settings.json:1]: also added another line in same file in latest commit as it was showing error for settings.json upon restarting the vscode.

Comment (beeme1mr) [reply .vscode/settings.json:1]: I don't think you should include this in the pr. It sounds like an issue with your local setup. Please remove this file.

**Review by beeme1mr — COMMENTED** [review]
Please remove the vscode config and we should be able to merge.

**Review by beeme1mr — APPROVED** [review]
